### PR TITLE
[Needs Review] B07902073 傅子芸

### PR DESCRIPTION
**Pull request by B07902073**
A Pull Request (PR) should explain why we need this PR and how you implement it, so others could understand it quickly.
Since this is just a homework, you do not need to explain why and how. 

**Do NOT change the status to NeedsReview if you do not want to be graded yet.**

Please fill in the following questions as feedback. It is totally optional and would not be graded.

## What is the most challenging part in this homework?
Using github 
## Is there any suggestion?
The autograder and flake8 are in different folders. This cause a bit trouble when testing some small changes frequently. 
<br />

### CODE REVIEW

**Comment by amjltc295** [comment]
What does `The autograder and flake8 are indifferent folders.` means? flake8 is not a folder.

File: src/students/B07902073.py [hunk 1]
```diff
@@ -0,0 +1,435 @@
...
+    from PIL import Image, ImageFont, ImageDraw
+    from logging_config import logger
+    img_url: str = 'https://i.imgur.com/B75zq0x.jpg'
+    r = request.urlopen(img_url)
+    data = r.read()
+    with open("1234.jpg", 'wb') as f:
+        f.write(data)
+    im = Image.open('1234.jpg')
+
+    def draw_text(
```
Comment (amjltc295) [inline src/students/B07902073.py]: Do not define draw_text() again.

Comment (B07902073) [reply src/students/B07902073.py]: Have removed it.

File: src/students/B07902073.py [hunk 2]
```diff
@@ -0,0 +1,416 @@
...
+
+    from PIL import Image, ImageFont, ImageDraw
+    from logging_config import logger
+    img_url: str = 'https://i.imgur.com/B75zq0x.jpg'
+    r = request.urlopen(img_url)
+    data = r.read()
+    with open("1234.jpg", 'wb') as f:
+        f.write(data)
+    result_img = Image.open('1234.jpg')
+    draw_text(result_img, "B07902073")
```
Comment (amjltc295) [inline src/students/B07902073.py]: Where is draw_text() from?

File: src/students/B07902073.py (hunk 1 above)
Comment (B07902073) [reply src/students/B07902073.py]: The flake8 file is under PythonHomework/  ,and the autograder is under PythonHomework/src/.  To run each file, I have to change direction between PythonHomework/ and PythonHomework/src/, right?

Comment (amjltc295) [reply src/students/B07902073.py]: Not exactly. You could do `flake8 students/<your student ID>.py` under `src`. Flake8 is not a file but a package. `.flake8` is its configuration file.

Comment (amjltc295) [reply src/students/B07902073.py]: You could also run the `autograder.py` under `PythonHomework/ `

Comment (amjltc295) [reply src/students/B07902073.py]: But I personally suggest using plugins that could help you check the Flake8 errors in your editor, [ALE](https://github.com/w0rp/ale) for examples.

Comment (B07902073) [reply src/students/B07902073.py]: Ok.  Got it.

File: src/students/B07902073.py (hunk 2 above)
Comment (B07902073) [reply src/students/B07902073.py]: Fixed. Is that right?

Comment (amjltc295) [reply src/students/B07902073.py]: If you have tested it, it should be correct.

Comment (B07902073) [reply src/students/B07902073.py]: @amjltc295

**Pull request merged**